packages/manager/apps/dedicated/client/app/ip/components/list/popover/ip.html

```diff
@@ -291,7 +291,33 @@
             ></button>
         </li>
 
-        <!-- IP DELETE -->
+        <!-- PCC DELETE -->
+        <li data-ng-if="ipBlock.type == PCC">
+            <button
+                type="button"
+                class="btn btn-link"
+                data-ng-click="setAction('ip/block/delete/ip-ip-block-delete', { ipBlock: ipBlock , tracking : tracking['delete-pcc']})"
+                data-oui-tooltip="{{ !ipBlock.canBeTerminated ? ('ip_failover_impossible_delete' | translate) : '' }}"
+                data-oui-tooltip-placement="left"
+                data-ng-disabled="!ipBlock.canBeTerminated"
+                data-ng-bind=":: ('ip_table_manage_delete_ipblock' | translate) + ADDITIONAL_IP"
+            ></button>
+        </li>
+
+        <!-- VRACK DELETE -->
+        <li data-ng-if="ipBlock.type == VRACK">
+            <button
+                type="button"
+                class="btn btn-link"
+                data-ng-click="setAction('ip/block/delete/ip-ip-block-delete', { ipBlock: ipBlock , tracking : tracking['delete-vrack']})"
+                data-oui-tooltip="{{ !ipBlock.canBeTerminated ? ('ip_failover_impossible_delete' | translate) : '' }}"
+                data-oui-tooltip-placement="left"
+                data-ng-disabled="!ipBlock.canBeTerminated"
+                data-ng-bind=":: ('ip_table_manage_delete_ipblock' | translate) + ADDITIONAL_IP"
+            ></button>
+        </li>
```

Comment:
**nitpick:** Both the buttons are similar except the scope set for `setAction`. So it can be combined.
```suggestion
        <li data-ng-if="ipBlock.type === PCC || ipBlock.type == VRACK">
            <button
                type="button"
                class="btn btn-link"
                data-ng-click="setAction('ip/block/delete/ip-ip-block-delete', { ipBlock: ipBlock , tracking : ipBlock.type == VRACK ? tracking['delete-vrack'] : tracking['delete-pcc']})"
                data-oui-tooltip="{{ !ipBlock.canBeTerminated ? ('ip_failover_impossible_delete' | translate) : '' }}"
                data-oui-tooltip-placement="left"
                data-ng-disabled="!ipBlock.canBeTerminated"
                data-ng-bind=":: ('ip_table_manage_delete_ipblock' | translate) + ADDITIONAL_IP"
            ></button>
        </li>
```

Reply:
Done Anoop